### ISSUE TITLE
[typescript][typings] Error with existing typings of version 1.2.0

### ISSUE THREAD

**Issue by mgruel** [auth0/node-jwks-rsa#15]
Observed behavior (tsc output):
```
Running "ts:build" (ts) task
Compiling...
Cleared fast compile cache for target: build
Using tsc v2.4.0
src/server/routes/token_authentication.ts(19,16): error TS2551: Property 'expressJwtSecret' does not exist on type 'typeof JwksRsa'. Did you mean 'ExpressJwtSecret'?

>> 1 non-emit-preventing type warning  
>> Error: tsc return code: 2
Warning: Task "ts:build" failed. Use --force to continue.

Aborted due to warnings.
```

Expected behavior:
Property exists on type JwksRsa

Example code (express middleware)
```javascript
import * as express from "express";
import * as jwt from "express-jwt";
import * as jwks from "jwks-rsa";

const AUTH0_DOMAIN = process.env.AUTH0_DOMAIN || "no-auth0-domain.available";

/**
 * Create base validation configuration for JWT created by Auth0
 */
const BASE_CONFIG: jwt.Options = {
  secret: jwks.expressJwtSecret({
    cache: true,
    rateLimit: true,
    jwksRequestsPerMinute: 5,
    jwksUri: `https://${AUTH0_DOMAIN}/.well-known/jwks.json`
  }),
  audience: "urn:operational_portal",
  issuer: `https://${AUTH0_DOMAIN}/`,
  algorithms: ["RS256"]
};

const CREDENTIALS_OPTIONAL_CONFIG: jwt.Options = Object.assign({}, BASE_CONFIG, {
  credentialsRequired: false
});

export function authenticationRequired(req: express.Request, res: express.Response, next: any) {
  return jwt(BASE_CONFIG)(req, res, next);
}

export function getUser(req: express.Request, res: express.Response, next: any) {
  return jwt(CREDENTIALS_OPTIONAL_CONFIG)(req, res, next);
}
```

**Comment by SMK1085** [comment]
Looks like there is a typo in the index.d.ts file on [line 34](https://github.com/auth0/node-jwks-rsa/blob/master/index.d.ts#L34), it is:
```
function ExpressJwtSecret(options: JwksRsa.Options): ExpressJwt.SecretCallback;
```
But instead it should be
```
function expressJwtSecret(options: JwksRsa.Options): ExpressJwt.SecretCallback;
```
to match the JS code. You can get a workaround by changing changing your TS code to
```
(<any>jwks).expressJwtSecret({
```
It's not clean, but at least you avoid the compiler error...

**Comment by jbreckmckye** [comment]
Until this is fixed, this module cannot be used in TypeScript projects.

**Comment by zacharynevin** [comment]
+1

**Comment by zacharynevin** [comment]
A potential hack around this could be the following:
```
(<any> jwks).expressJwtSecret({...})
```

**Comment by sandrinodimattia** [comment]
This was released as 1.2.1 from #25
